packages/tailwind/src/_docs/pages/working-with-tailwind.stories.tsx

```diff
@@ -1,14 +1,14 @@
 import React from "react"
 import { StoryFn } from "@storybook/react"
-import { Card } from "~components/Card"
-import { Heading } from "~components/Heading"
 import {
+  Card,
   CustomSurvey,
   CustomUnattributedSurvey,
-} from "~components/Illustration"
-import { Skirt } from "~components/Skirt"
-import { Button } from "~components/__actions__/v2"
-import { TailwindStoryTemplate } from "~storybook/components/TailwindStoryTemplate"
+  Skirt,
+  Heading,
+} from "@kaizen/components"
+import { Button } from "@kaizen/components/v2/actions"
```

Comment:
Ah, I recall this caused funny errors with alias at first. Perhaps a devDep may be safe enough? We'd need to investigate.